### PR TITLE
List screens default sort by id

### PR DESCRIPTION
**Pull request by snimavat**
https://github.com/9ci/domain9/issues/2280

### CODE REVIEW

File: gorm-rest/src/main/groovy/yakworks/rest/gorm/controller/EntityResponder.groovy [hunk 1]
```diff
@@ -114,6 +114,9 @@ class EntityResponder<D> {
         // pclone.removeAll {it.key in whitelistKeys }
         try {
             QueryArgs qargs = QueryArgs.of(pager)
+            if(!qargs.sort) {
```
Comment (basejump) [inline gorm-rest/src/main/groovy/yakworks/rest/gorm/controller/EntityResponder.groovy:117]: dig into this more. See notes below. 

```groovy
//XXX @SUD why dont we build in QueryArgs.of(pager)? we only build in QueryArgs.of(params). explain here and in QueryArgs.of(pager)
//also will this effet the sort
QueryArgs qargs = QueryArgs.of(pager)
// when paging we need a sort so rows dont show up next page, so use id if nothing is specified.
// see https://github.com/9ci/domain9/issues/2280
```

your checking sort, but it might be in the q too right? as `$sort`? nav through this more and check the edge cases. Also, I dont think this belongs here. should be centralized in the QueryArgs logic with good tests.

**Review by basejump — CHANGES_REQUESTED** [review]

File: gorm-rest/src/main/groovy/yakworks/rest/gorm/controller/EntityResponder.groovy (hunk 1 above)
Comment (snimavat) [reply gorm-rest/src/main/groovy/yakworks/rest/gorm/controller/EntityResponder.groovy:117]: @basejump if we put it in QArgs logic, it will start affecting every time the query is executed, including "tests"
We want this logic only for list screens on UI, isnt it ?
